src/pyodide/python-entrypoint-helper.js

```diff
@@ -153,16 +154,26 @@ async function setupPackages(pyodide) {
       if (!EMBEDDED_PYTHON_PACKAGES.includes(name)) {
         pythonRequirements.push(name);
       }
+      // Packages are not embedded in workerd.
+      // TODO: Improve package loading in workerd.
+      if (isWorkerd) {
+        micropipRequirements.push(name);
+      }
     }
   }
 
+  if (isWorkerd) {
+    pythonRequirements.push("micropip");
+  }
```

Comment:
Perhaps instead of `empty` we should stick micropip and packaging in there. It would significantly improve load time when micropip is needed and I don't think they are very big. But this can happen in a followup.